### PR TITLE
Patch Datastore and add mock profiles on server

### PR DESCRIPTION
**Pull request by AndrewBoya**
The way I implemented these mock profiles is not my proudest work, but it is functional for Jonathan's code walk this afternoon.

### CODE REVIEW

File: backend/src/main/java/datastorage/SimpleDataStore.java
Comment (JMS100) [inline backend/src/main/java/datastorage/SimpleDataStore.java:1]: Good catch!

File: backend/src/main/java/server/Server.java
Comment (JMS100) [inline backend/src/main/java/server/Server.java:1]: We might have to review, but looks good for codewalk!

**Review by JMS100 — APPROVED** [review]

**Review by jonathanzhou1 — APPROVED** [review]
Josh Joshington about to pop off with the most insane stats ever

File: backend/src/main/java/server/Server.java
Comment (AndrewBoya) [reply backend/src/main/java/server/Server.java:1]: Yeah definitely. I wanted to produce something that Jonathan could have for ease of demonstration during his code walks, but I definitely want to refactor this for the final demonstration.

**Pull request merged**